src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -43,6 +52,55 @@ export const ALERT_TYPES = {
   droHearingNoShow: 'dro_hearing_no_show',
 };
 
+/**
+ * Takes an array of appeals and returns another array of issue descriptions
+ * and where in the appeal lifecycle each issue is (open, remand, granted, denied)
+ * @typedef {Object} issue an individual issue - many issues can be a part of a single appeal
+ * @property {bool} active indicates whether an appeal is open or closed
+ * @property {string} description more info about the specific injury in the issue
+ * @property {string} diagnosticCode a codified version of the description
+ * @property {('field_grant'|'withdrawn'|'allowed'|'denied'|'remand'|'cavc_remand')} lastAction
+ * @property {string} date TO-DO: unsure of what this date siginifies
+ * ------------------------------------------------------------------------------------------------
+ * @typedef {Object} segmentedIssue issue with descriptor and status information
+ * @property {('granted'|'remand'|'allowed'|'denied')} status lifecycle stage of an issue
+ * @property {string} description pass-through for the description info of passed in issue object
+ * ------------------------------------------------------------------------------------------------
+ * @param {issue[]} issues all the individual issues that are attached to an appeal
+ * @returns {segmentedIssue[]} an array of issue objects with stuses and descriptions
+ */
+export function categorizeIssues(issues) {
+  return issues.map((issue) => {
+    let status = '';
+    switch (issue.lastAction) {
+      case ISSUE_STATUS.fieldGrant:
+        status = 'granted';
+        break;
+      case ISSUE_STATUS.withdrawn:
+        status = 'denied';
```

Comment:
Actually, we need a withdrawn section, which should go underneath the granted and denied sections. The rest of this mapping is correct.

Reply:
Ahh, I was kind of unsure about whether 'withdrawn' is a separate section. Makes sense 👍

Reply:
See new screenshot, I think the 'withdrawn' section is good to go now.

Reply:
:+1:

Reply:
The new screenshot looks good, but I'm not seeing where that's at in the code. Did you push it up?

Reply:
Not only did I not push it up, but I can't find it locally. Not sure what I did to lose it all but it's pushed up now.